extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -107,7 +111,8 @@ public function testRenderInfoboxDataProvider() {
 			[
 				'input' => [ ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "input" 17 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)